orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/LogoutService.java

```diff
@@ -137,66 +143,54 @@ public APIGatewayProxyResponseEvent generateDefaultLogoutResponse(
             cloudwatchMetricsService.incrementLogout(clientId);
         }
         return generateLogoutResponse(
-                configurationService.getDefaultLogoutURI(),
+                frontEndPages.logoutURI(Optional.empty()),
                 state,
-                Optional.empty(),
                 auditUser,
                 clientId,
                 rpPairwiseId);
     }
 
-    public APIGatewayProxyResponseEvent generateLogoutResponse(
+    public APIGatewayProxyResponseEvent generateCustomLogoutResponse(
             URI logoutUri,
             Optional<String> state,
-            Optional<ErrorObject> errorObject,
             TxmaAuditUser auditUser,
-            Optional<String> clientId) {
-        return generateLogoutResponse(
-                logoutUri, state, errorObject, auditUser, clientId, Optional.empty());
+            Optional<String> clientId,
+            Optional<String> rpPairwiseId) {
```

Comment:
I think we're fighting against each other here 😄 I introduced this overload very recently. I'd like to move away from Optional as a parameter in general, but that was too large of a change for the change I was working on. 

As a general pattern I think we should be using overloads for truly optional parameters, but open to discussion on that.